.infra/index.ts

```diff
@@ -258,7 +258,6 @@ if (isAdhocEnv) {
 } else {
   appsArgs = [
     {
-      port: 3000,
```

Comment:
This is replace with `ports` prop below?

Reply:
Yeah. `port` is "deprecated", and internally in pulumi-common, `port` translates to `ports`
https://github.com/dailydotdev/pulumi-common/blob/6845b5d4e62d30e0c9b6efb9a8beb7d62a5404cd/src/suite/index.ts#L477-L479